install_config/configure_kerberos_for_sso.adoc

```diff
@@ -0,0 +1,174 @@
+==== Configuring an Apache proxy or Single Sign On authentication via Kerberos
+
+Configuring SSO authentication makes life much easier for users that are
+enrolled in a Windows Active Directory domain or a Linux IPA domain. Both these
+enteprise authentication systems use Kerberos and that allows a seamless
+experience where the user doesn't need to use additional passwords to
+authenticate to systems and services.
+
+The key challenge in getting this setup to work properly is correct
+configuration of domains names. Kerberos realies on correct DNS resolution and
+fully qualified domain names to resolve how to acquire the proper token to
+authenticate to a network service. Naming errors are the leading cause of
+unsuccessful deployments.
+
+===== Installation prerequisites
+
+First establish the (public) DNS name that you are going to assign to the
```

Comment:
Are there any other limitations that a user needs to know about?

Reply:
Nothing comes to mind.